public/index.html

```diff
@@ -10,6 +10,7 @@
   <link rel="preconnect" href="https://api.scrivito.com" crossorigin>
   <link rel="preconnect" href="https://api.scrivito.com">
   <link rel="stylesheet" href="/index.css">
+  <link rel="stylesheet" href="/node_modules/aos/dist/aos.css">
```

Comment:
Please not not import this here. Instead include it into the `index.scss` similar to how the css of slick is included (e.g. https://github.com/Scrivito/scrivito_example_app_js/blob/master/src/assets/stylesheets/index.scss#L47).